### PR TITLE
[Docs] Update docs/zh_cn/tutorials/customize_config.md

### PR DESCRIPTION
**Pull request by RangeKing**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Help build Chinese Documentation.

## Modification
1. Update the zh_cn docs `customize_config.md` and `get_started.md`.
2. Fix some typos and grammatical errors in en docs `customize_config.md` and `get_started.md`.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by zytx121** [comment]
Hi @RangeKing 
`get_started.md`  has been translated by others. https://github.com/open-mmlab/mmrotate/pull/11
Please submit `customize_config.md` separately.

File: docs/zh_cn/tutorials/customize_config.md [hunk 1]
```diff
@@ -1,185 +1,180 @@
-# Tutorial 1: Learn about Configs
+# 教程1：学习配置文件
```
Comment (jbwang1997) [inline docs/zh_cn/tutorials/customize_config.md:1]: 教程 1

File: docs/zh_cn/tutorials/customize_config.md [hunk 2]
```diff
@@ -1,185 +1,180 @@
...
 
 
-## Modify a config through script arguments
+## 通过脚本参数修改配置
 
-When submitting jobs using "tools/train.py" or "tools/test.py", you may specify `--cfg-options` to in-place modify the config.
+当运行 `tools/train.py` 或者 `tools/test.py` 时，可以通过 `--cfg-options` 来修改配置。
 
-- Update config keys of dict chains.
+- 更新字典链的配置键
```
Comment (jbwang1997) [inline docs/zh_cn/tutorials/customize_config.md:13]: 可以改成：更新字典链中的配置

File: docs/zh_cn/tutorials/customize_config.md [hunk 3]
```diff
@@ -1,185 +1,180 @@
...
+        type='RotatedRetinaHead',# bbox_head 的类型是 'RRetinaHead'
+        num_classes=15,  # 分类的类别数量
+        in_channels=256,  # bbox head 输入通道数
+        stacked_convs=4,  # head 卷积层的层数
+        feat_channels=256,  # head 卷积层的特征通道
+        assign_by_circumhbbox='oc',  # obb2hbb 的旋转定义方式
+        anchor_generator=dict(  # 锚点(Anchor)生成器的配置
+            type='RotatedAnchorGenerator',  # 锚点生成器类别
             octave_base_scale=4,  # The base scale of octave.
             scales_per_octave=3,  #  Number of scales for each octave.
```
Comment (jbwang1997) [inline docs/zh_cn/tutorials/customize_config.md:85]: 这里没有翻译，octave 可以直接使用。

Comment (RangeKing) [reply docs/zh_cn/tutorials/customize_config.md]: 已参考 OpenMMLab [知乎文章](https://zhuanlan.zhihu.com/p/346198300) 翻译

**Comment by RangeKing** [comment]
> Hi @RangeKing `get_started.md` has been translated by others. #11 Please submit `customize_config.md` separately.

@zytx121 Already revert `get_started.md` related commits.

**Comment by jbwang1997** [comment]
Thank you for your contributions. Please follow https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md to set pre-commit and fix the lint error.

**Comment by RangeKing** [comment]
> Thank you for your contributions. Please follow https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md to set pre-commit and fix the lint error.

@jbwang1997 Already fixed the lint error

**Review by jbwang1997 — APPROVED** [review]
Look good to me. Kindly ping @zytx121 to have a look.

File: docs/zh_cn/tutorials/customize_config.md [hunk 4]
```diff
@@ -1,185 +1,179 @@
...
-            type='BN',  # Type of norm layer, usually it is BN or GN
-            requires_grad=True),  # Whether to train the gamma and beta in BN
-        norm_eval=True,  # Whether to freeze the statistics in BN
-        style='pytorch',  # The style of backbone, 'pytorch' means that stride 2 layers are in 3x3 conv, 'caffe' means stride 2 layers are in 1x1 convs.
-        init_cfg=dict(type='Pretrained', checkpoint='torchvision://resnet50')),  # The ImageNet pretrained backbone to be loaded
+    type='RotatedRetinaNet',  # 检测器(detector)名称
+    backbone=dict(  # 主干网络的配置文件
+        type='ResNet',  # # 主干网络的类别
+        depth=50,  # 主干网络的深度
+        num_stages=4,  # 主干网络状态(stages)的数目
```
Comment (ZwwWayne) [inline docs/zh_cn/tutorials/customize_config.md:57]: 状态 -> 阶段

File: docs/zh_cn/tutorials/customize_config.md [hunk 5]
```diff
@@ -1,185 +1,179 @@
...
-        out_channels=256,  # The output channels of each level of the pyramid feature map
-        start_level=1,  # Index of the start input backbone level used to build the feature pyramid
-        add_extra_convs='on_input',  # It specifies the source feature map of the extra convs
-        num_outs=5),  # The number of output scales
+        type='FPN',  # 检测器的 neck 是 FPN， 我们同样支持 'ReFPN'
+        in_channels=[256, 512, 1024, 2048],  # 输入通道数，这与主干网络的输出通道一致
+        out_channels=256,  # 金字塔特征图每一层的输出通道
+        start_level=1,  # 用于构建特征金字塔的主干网络起始输入层索引值
+        add_extra_convs='on_input',  # 决定是否在原始特征图之上添加卷积层
+        num_outs=5),  # 输出的范围(scales)
```
Comment (ZwwWayne) [inline docs/zh_cn/tutorials/customize_config.md:73]: 决定输出多少个尺度的特征图

File: docs/zh_cn/tutorials/customize_config.md [hunk 6]
```diff
@@ -1,185 +1,179 @@
...
+dataset_type = 'DOTADataset'  # 数据集类型，这将被用来定义数据集
+data_root = '../datasets/split_1024_dota1_0/'  # 数据的根路径
+img_norm_cfg = dict(  # 图像归一化配置，用来归一化输入的图像
+    mean=[123.675, 116.28, 103.53],  # 预训练里用于预训练主干网络模型的平均值
+    std=[58.395, 57.12, 57.375],  # 预训练里用于预训练主干网络模型的标准差
+    to_rgb=True)  # 预训练里用于预训练主干网络的图像的通道顺序
+train_pipeline = [  # 训练流程
+    dict(type='LoadImageFromFile'),  # 第 1 个流程，从文件路径里加载图像
+    dict(type='LoadAnnotations',  # 第 2 个流程，对于当前图像，加载它的注释信息
+         with_bbox=True),  # 是否使用标注框(bounding box)， 目标检测需要设置为 True
```
Comment (ZwwWayne) [inline docs/zh_cn/tutorials/customize_config.md:130]: 是否使用 -> 是否加载

**Review by ZwwWayne — APPROVED** [review]

**Review by zytx121 — APPROVED** [review]
LGTM

**Comment by ZwwWayne** [comment]
Hi @RangeKing ,
Thanks for your kind PR, could you help resolve the comments so that we can merge this PR?

**Comment by RangeKing** [comment]
> Hi @RangeKing , Thanks for your kind PR, could you help resolve the comments so that we can merge this PR?

Resolved.
I refer to these code comments from [MMDetection Tutorial](https://mmdetection.readthedocs.io/zh_CN/v2.19.1/tutorials/config.html). Need to make the same changes to that tutorial?

**Pull request merged**